docs/en_US/getting-started/getting-started.md

```diff
@@ -38,6 +38,31 @@ $ curl -f -L "https://github.com/emqx/emqx-operator/releases/download/1.1.5/emqx
       --namespace emqx-operator-system \
       --create-namespace
    ```
+**Note**:
+If you want to collect metrics about `EMQX Operator` by `Prometheus`,first of all you need to deploy [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) and apply the application of `ServiceMonitor`
```

Comment:
```suggestion
If you want to collect metrics about `EMQX Operator` by `Prometheus`,first of all you need to deploy [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator)
```

I think "deploy Prometheus Operator" is enough, because `ServiceMonitor ` is a part of Prometheus Operator, and `ServiceMonitor` is not application, it's CRD